### PR TITLE
Remove VXLAN to Ether binding

### LINKED ISSUE #1674: VXLAN Binding to IPv6 or IPv4 doesn't work
#### Brief description
When doing a VXLAN()/IP or VXLAN()/IPv6 the binding does not work properly. Both IP layers are interpreted as Ethernet Layers.

#### Environment

-   Scapy version: `2.4.0`
-   Python version: `3.5.2`
-   Operating System: `Ubuntu 16.04`

#### How to reproduce
```python
>>> issue = Ether()/IP()/UDP()/VXLAN(flags=4, NextProtocol=1)/IP()
>>> issue.show2()
###[ Ethernet ]### 
  dst= ff:ff:ff:ff:ff:ff
  src= 00:00:00:00:00:00
  type= 0x800
###[ IP ]### 
     version= 4
     ihl= 5
     tos= 0x0
     len= 56
     id= 1
     flags= 
     frag= 0
     ttl= 64
     proto= udp
     chksum= 0x7cb2
     src= 127.0.0.1
     dst= 127.0.0.1
     \options\
###[ UDP ]### 
        sport= 4789
        dport= 4789
        len= 36
        chksum= 0xd838
###[ VXLAN ]### 
           flags= NextProtocol
           reserved0= 0
           NextProtocol= IPv4
           vni= 0x0
           reserved2= 0x0
###[ Ethernet ]### 
              dst= 45:00:00:14:00:01
              src= 00:00:40:00:7c:e7
              type= 0x7f00
###[ Raw ]### 
                 load= '\x00\x01\x7f\x00\x00\x01'
```

#### Actual result
```python
###[ Ethernet ]### 
  dst= ff:ff:ff:ff:ff:ff
  src= 00:00:00:00:00:00
  type= 0x800
###[ IP ]### 
     version= 4
     ihl= 5
     tos= 0x0
     len= 56
     id= 1
     flags= 
     frag= 0
     ttl= 64
     proto= udp
     chksum= 0x7cb2
     src= 127.0.0.1
     dst= 127.0.0.1
     \options\
###[ UDP ]### 
        sport= 4789
        dport= 4789
        len= 36
        chksum= 0xd838
###[ VXLAN ]### 
           flags= NextProtocol
           reserved0= 0
           NextProtocol= IPv4
           vni= 0x0
           reserved2= 0x0
###[ Ethernet ]### 
              dst= 45:00:00:14:00:01
              src= 00:00:40:00:7c:e7
              type= 0x7f00
###[ Raw ]### 
                 load= '\x00\x01\x7f\x00\x00\x01'
```

#### Expected result
```python
###[ Ethernet ]### 
  dst= ff:ff:ff:ff:ff:ff
  src= 00:00:00:00:00:00
  type= 0x800
###[ IP ]### 
     version= 4
     ihl= 5
     tos= 0x0
     len= 56
     id= 1
     flags= 
     frag= 0
     ttl= 64
     proto= udp
     chksum= 0x7cb2
     src= 127.0.0.1
     dst= 127.0.0.1
     \options\
###[ UDP ]### 
        sport= 4789
        dport= 4789
        len= 36
        chksum= 0xd838
###[ VXLAN ]### 
           flags= NextProtocol
           reserved0= 0
           NextProtocol= IPv4
           vni= 0x0
           reserved2= 0x0
###[ IP ]### 
              version= 4
              ihl= 5
              tos= 0x0
              len= 20
              id= 1
              flags= 
              frag= 0
              ttl= 64
              proto= hopopt
              chksum= 0x7ce7
              src= 127.0.0.1
              dst= 127.0.0.1
              \options\
```

### PR DESCRIPTION
**Pull request by StrydeAlpha**
fixes #1674

### CODE REVIEW

File: scapy/layers/vxlan.py [hunk 1]
```diff
@@ -85,7 +85,6 @@ def mysummary(self):
 # By default, set both ports to the RFC standard
 bind_layers(UDP, VXLAN, sport=4789, dport=4789)
 
-bind_layers(VXLAN, Ether)
```
Comment (p-l-) [inline scapy/layers/vxlan.py]: We should keep at least this:
```suggestion
bind_layers(VXLAN, Ether, NextProtocol=0)
```

File: scapy/layers/vxlan.py [hunk 2]
```diff
@@ -85,7 +85,6 @@ def mysummary(self):
...
 bind_layers(VXLAN, IP, NextProtocol=1)
 bind_layers(VXLAN, IPv6, NextProtocol=2)
 bind_layers(VXLAN, Ether, flags=4, NextProtocol=0)
```
Comment (gpotter2) [inline scapy/layers/vxlan.py]: @p-l- So this comes in useless ?

File: scapy/layers/vxlan.py (hunk 1 above)
Comment (StrydeAlpha) [reply scapy/layers/vxlan.py]: Ok, I'll add some tests for the fix, I hope to have it ready by tomorrow

File: scapy/layers/vxlan.py (hunk 2 above)
Comment (p-l-) [reply scapy/layers/vxlan.py]: Nope, this sets `flags=4` when building `VXLAN() / Ether()`.

**Comment by p-l-** [comment]
Also, FYI, this PR breaks the current tests.

**Comment by StrydeAlpha** [comment]
I moved to PR https://github.com/secdev/scapy/pull/1682 so I can manage the PR in a personal branch

**Pull request closed without merging**